tests/dummy/app/pods/demo/template.hbs

```diff
@@ -17,7 +17,7 @@
       &lt;placeholder: controls&gt;
   {{/block-slot}}
   {{#block-slot 'actions' as |action|}}
-    {{action.button icon='frost/infobar-create' text='Create' onClick=(action 'createUser')}}
+    {{action.button icon='frost/infobar-create' text='Click me!' onClick=(action 'triggerAction')}}
```

Comment:
Very minor, but I'd like to start consistently line-wrapping the properties - e.g.

```
{{action.button
  icon='infobar-create'
  text='Click me!'
  onClick=(action 'triggerAction')
}}
```